components/esp_modem/test/target/main/pppd_test.cpp

```diff
@@ -1,5 +1,5 @@
 /*
- * SPDX-FileCopyrightText: 2022-2024 Espressif Systems (Shanghai) CO LTD
+ * SPDX-FileCopyrightText: 2022-2023 Espressif Systems (Shanghai) CO LTD
```

Comment:
Are we going back in time :)?

Reply:
Thanks for noticing, fixed!

> Are we going back in time :)?

We are, actually. Created this commits few days back, but had to temporarily modify the test to merge URC before investigating target test failures + wanted to rule out the catch2 suspect :-)
Then, after reverting to this commit again, the copyright changed back and no pre-commit hook nor the CI ever noticed :shrug: